src/views/PropTableView.vue

```diff
@@ -473,37 +556,51 @@ onBeforeMount(() => {
+        // find the current/default profile
+        defaultProfile.value = ui.profiles[profiles.value.find(p => p.default)!.uri];
+        
+        // if specify mediatype, or profile is not default or alt, redirect to API
+        if ((route.query && route.query._profile) &&
+            (route.query._mediatype || ![defaultProfile.value.token, ALT_PROFILES_TOKEN].includes(route.query._profile as string))) {
+                window.location.replace(`${apiBaseUrl}${route.path}?_profile=${route.query._profile}${route.query._mediatype ? `&_mediatype=${route.query._mediatype}` : ""}`);
+        }
+
+        // disable right nav if AltView
+        if (isAltView.value) {
+            ui.rightNavConfig = { enabled: false };
+        } else {
+            ui.rightNavConfig = { enabled: true, profiles: profiles.value, currentUrl: route.path };
+        }
+
+        parseIntoStore(data.value);
+        getProperties();
+        if (!isAltView.value && childrenConfig.value.showChildren) {
+            getChildren();
+        }
+
+        document.title = item.value.title ? `${item.value.title} | Prez` : "Prez";
+        ui.breadcrumbs = getBreadcrumbs();
+    });
+}
+
 onMounted(() => {
     loading.value = true;
     // wait for profiles to be set in Pinia
     ensureProfiles().then(() => {
         console.log("profiles ready")
-        doRequest(`${apiBaseUrl}${route.fullPath}`, () => {
-            // find the current/default profile
-            defaultProfile.value = ui.profiles[profiles.value.find(p => p.default)!.uri];
-            
-            // if specify mediatype, or profile is not default or alt, redirect to API
-            if ((route.query && route.query._profile) &&
-                (route.query._mediatype || ![defaultProfile.value.token, ALT_PROFILES_TOKEN].includes(route.query._profile as string))) {
-                    window.location.replace(`${apiBaseUrl}${route.path}?_profile=${route.query._profile}${route.query._mediatype ? `&_mediatype=${route.query._mediatype}` : ""}`);
-            }
-
-            // disable right nav if AltView
-            if (isAltView.value) {
-                ui.rightNavConfig = { enabled: false };
-            } else {
-                ui.rightNavConfig = { enabled: true, profiles: profiles.value, currentUrl: route.path };
-            }
-
-            parseIntoStore(data.value);
-            getProperties();
-            if (!isAltView.value && childrenConfig.value.showChildren) {
-                getChildren();
-            }
 
-            document.title = item.value.title ? `${item.value.title} | Prez` : "Prez";
-            ui.breadcrumbs = getBreadcrumbs();
-        });
+        if (item.value.baseClass === qnameToIri("skos:ConceptScheme")) {
+            countDoRequest(`${apiBaseUrl}/count?curie=${route.path.split("/").slice(-1)[0]}&inbound=${encodeURIComponent(qnameToIri("skos:inScheme"))}`, () => {
+                if (parseInt(countData.value.replace('"', "")) <= conceptPerPage) {
```

Comment:
Ah, this is a mistake in the API. I think it is returning as JSON that's why the value has quotation marks around it. I'll set it to a plain text response.

Reply:
Fixed in https://github.com/RDFLib/prez/pull/134/commits/6953a30a86d09f74f20896d7ba4d81bc1e5a2461.